apps/site/src/app/(home)/sections/Landing/Landing.tsx

```diff
@@ -1,26 +1,35 @@
 "use client";
 
-import Button from "@/lib/components/Button/Button";
-import styles from "./Landing.module.css";
-
 import { Suspense } from "react";
 import { PerspectiveCamera } from "@react-three/drei";
+
 import { View } from "@/components/canvas/View";
+import Button from "@/lib/components/Button/Button";
 import Fireflies from "../../components/Fireflies";
 
+import landingBackground from "@/assets/backgrounds/landing-background.jpg";
+
 const Landing = () => {
 	return (
-		<section className={styles.landingSection}>
+		<section
+			style={{ backgroundImage: `url(${landingBackground.src})` }}
+			className="bg-auto xl:bg-cover bg-no-repeat bg-center"
```

Comment:
Issue: the background image does not scale down enough on shorter views.

Reply:
Changed it so `bg-cover` is now used for all viewport sizes.